src/app/open-source/page.tsx

```diff
@@ -19,23 +22,68 @@ export default function OpenSourcePage() {
                 </div>
                 <div className="container mx-auto px-4">
                     <section className="mb-8">
-                        <h2 className="mb-4 text-2xl font-bold">Who is the Open Source Team?</h2>
-                        <p className="text-lg md:text-xl">
-                            The CS Club Open Source Team is a place for creative and inventive
-                            students who want to build innovative open-source software together. We
-                            are a new team currently working on a few projects, but we have many
-                            more exciting projects planned for the future!
-                        </p>
-                    </section>
-                    <section className="mb-8 flex justify-center text-black">
-                        <Button
-                            type="button"
-                            colour="orange"
-                            width="w-full"
-                            href="https://docs.google.com/forms/d/e/1FAIpQLSe2uvnn4qW95yJ7TzyDo7QMjzhkawvdERRDmeBLMwloi-nqjg/viewform"
-                        >
-                            Join Us Now
-                        </Button>
+                        <div className="grid grid-cols-1 gap-8 lg:grid-cols-2">
```

Comment:
Fix the overlap at about 3/4 screen width by making the image go below the "Join the Open Source Team!" earlier (between `xl` and `2xl`). 

![image](https://github.com/user-attachments/assets/17f824a0-19ed-449b-a1ae-9fbfa0c877ed)